crates/ruff/src/rules/wemake_python_styleguide/helpers/naming.rs

```diff
@@ -0,0 +1,78 @@
+use std::path::Path;
+
+use once_cell::sync::Lazy;
+use regex::Regex;
+
+const ALIAS_NAMES_WHITELIST: [&str; 7] = ["np", "pd", "df", "plt", "sns", "tf", "cv"];
+
+const UNUSED_PLACEHOLDER: char = '_';
+
+static UNUSED_VARIABLE_REGEX: Lazy<Regex> = Lazy::new(|| Regex::new(r"^_+$").unwrap());
+
+/// Checks whether the given ``name`` is unused.
+fn is_unused(name: &str) -> bool {
+    UNUSED_VARIABLE_REGEX.is_match(name)
+}
```

Comment:
We try to avoid regex whenever possible because it is significantly slower than manual string operations. What are we testing here, whether the variable name starts with a `?`. We can get the same result with `name.starts_with('_')`

Reply:
I think in the original implementation the `trim=True` flag is meant to trim underscores from start and end (see [wemake_python_styleguide/logic/naming/logical.py#L103](https://github.com/wemake-services/wemake-python-styleguide/blob/a710b5f18ea35a7a09b713cc86cabc5a4fc6b25d/wemake_python_styleguide/logic/naming/logical.py#L103)). For example `z_` and `__z` should both be detected as too short variables names.

Reply:
fixup in 197c4ce99d1c3387e95343306063bcedc6f0b6be